Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apt-get update && apt-get install -y \
     libssl-dev \
     logrotate \
     sudo \
+    vim \
```

Comment:
Not sure if you want to keep that or not, I just find it handy when troubleshooting, and it does not add much girth to the image.